### PR TITLE
ref(ts): Convert OrganizationSettingsLayout to typescript

### PR DESCRIPTION
**Pull request by matejminar**

### CODE REVIEW

**Review by markstory — APPROVED** [review]

File: src/sentry/static/sentry/app/views/settings/organization/organizationSettingsLayout.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import {RouteComponentProps} from 'react-router/lib/Router';
+
+import OrganizationSettingsNavigation from 'app/views/settings/organization/organizationSettingsNavigation';
+import SettingsLayout from 'app/views/settings/components/settingsLayout';
+
+type Props = RouteComponentProps<{orgId: string}, {}> & {
+  children: React.ReactNode;
+};
+
+function OrganizationSettingsLayout(props: Props) {
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/views/settings/organization/organizationSettingsLayout.tsx:11]:
```suggestion
function OrganizationSettingsLayout({children, ...props}: Props) {
```

File: src/sentry/static/sentry/app/views/settings/organization/organizationSettingsLayout.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  return (
+    <SettingsLayout
+      {...props}
+      renderNavigation={() => <OrganizationSettingsNavigation {...props} />}
+    >
+      {props.children}
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/views/settings/organization/organizationSettingsLayout.tsx:17]:
```suggestion
      {children}
```

File: src/sentry/static/sentry/app/views/settings/organization/organizationSettingsLayout.tsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+type Props = RouteComponentProps<{orgId: string}, {}> & {
+  children: React.ReactNode;
+};
+
+function OrganizationSettingsLayout(props: Props) {
+  return (
+    <SettingsLayout
+      {...props}
+      renderNavigation={() => <OrganizationSettingsNavigation {...props} />}
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/views/settings/organization/organizationSettingsLayout.tsx:15]: Please, is it necessary to forward the children here?

Comment (matejminar) [reply src/sentry/static/sentry/app/views/settings/organization/organizationSettingsLayout.tsx]: I think not, I simplified it a bit 👍

**Review by priscilawebdev — APPROVED** [review]

**Pull request merged**